chapter-04-scim-user-migration/deployment/README.md

```diff
@@ -13,15 +16,29 @@ docker pull curity.azurecr.io/curity/idsvr
 docker compose --project-name scim up
 ```
 
-## Do the Initial Setup
+## Get a License File for the Curity Identity Server
+
+Run the following script from the root folder of this GitHub repository.\
```

Comment:
In my local preview, it renders the escape character at the end of the line. Please remove.

Reply:
```suggestion
Run the following script from the root folder of this GitHub repository.
```